fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/AsyncWebServiceRequestHandlerSpec.groovy

```diff
@@ -42,6 +42,7 @@ class AsyncWebServiceRequestHandlerSpec extends Specification {
         1 * dws.postDruidQuery(rc, _, null, null, groupByQuery) >> { a0, a1, a2, a3, a4 ->
             // Save the success callback
             sc = a1
+            return null
```

Comment:
@archolewa this method (`postDruidQuery`) needs to return _something_, but it should be able to return just about anything, since nothing in this test is looking at the return value. Given that, `sc = a1` _should_ have some sort of return value and should be OK. _If_ that's not true, then having `return null` here is fine I think.